bin/dmarc_send_reports

```diff
@@ -279,68 +279,85 @@ sub email {
     my $rid;
     $rid = $$agg_ref->metadata->report_id if $agg_ref;
 
+    my $sendmail_binary = $report->config->{sendmail}->{binary} || q{};
```

Comment:
for empty strings, I much prefer using `''` syntax over the many other ways perl offers. It's more idiomatic (especially for non-perl programmers) and it's recommended in _Perl Best Practices_.